Packs/MailSender_LongRunning/Integrations/MailSenderLongRunning/MailSenderLongRunning.yml

```
@@ -0,0 +1,137 @@
+      name: htmlBody
+    - description: Address that should be used to reply to the message
+      name: replyTo
+    - description: A comma-separated list of IDs of war room entries that contain
+        the files that should be attached to the email
+      isArray: true
+      name: attachIDs
+    - description: A comma-separated list to rename file-names of corresponding attachments
+        IDs. (e.g. rename first two files - attachNames=file_name1,file_name2. rename
+        first and third file - attachNames=file_name1,,file_name3)
+      isArray: true
+      name: attachNames
+    - description: A comma-separated list of CIDs to embed attachments inside the
+        email itself
+      isArray: true
+      name: attachCIDs
+    - description: Desired name for attached file. Multiple files are supported as
+        comma-separated list. (e.g. transientFile="t1.txt,temp.txt,t3.txt" transientFileContent="test
+        2,temporary file content,third file content" transientFileCID="t1.txt@xxx.yyy,t2.txt@xxx.zzz")
+      isArray: true
+      name: transientFile
+    - description: Content for attached file. Multiple files are supported as comma-separated
+        list. (e.g. transientFile="t1.txt,temp.txt,t3.txt" transientFileContent="test
+        2,temporary file content,third file content" transientFileCID="t1.txt@xxx.yyy,t2.txt@xxx.zzz")
+      isArray: true
+      name: transientFileContent
+    - description: CID for attached file if we want it inline. Multiple files are
+        supported as comma-separated list. (e.g. transientFile="t1.txt,temp.txt,t3.txt"
+        transientFileContent="test 2,temporary file content,third file content" transientFileCID="t1.txt@xxx.yyy,t2.txt@xxx.zzz")
+      isArray: true
+      name: transientFileCID
+    - description: 'Replace {varname} variables with values from this argument. Expected
+        values are in the form of a JSON document like {"varname": {"value": "some
+        value", "key": "context key"}}. Each var name can either be provided with
+        the value or a context key to retrieve the value from.'
+      name: templateParams
+    - description: 'A CSV list of additional headers in the format: headerName=headerValue.
+        For example: "headerName1=headerValue1,headerName2=headerValue2".'
+      isArray: true
+      name: additionalHeader
+    - description: Raw email message to send. If provided, all other arguments, but
+        to, cc and bcc, will be ignored.
+      name: raw_message
+    - auto: PREDEFINED
+      defaultValue: "no"
+      description: Add email to outbox queue to be sent asynchronously
+      name: async
+      predefined:
+      - "yes"
+      - "no"
+    - description: Entitlement to handle when async email is sent
+      name: entitlement
+    description: Send an email
+    name: send-mail
+  dockerimage: demisto/python:2.7.18.10627
+  longRunning: true
+  runonce: false
+  script: ''
+  subtype: python2
```

Make sure the code is compatible with py3 since we don't support py2.
```suggestion
  subtype: python3
```